docs/development/updating_ossec.rst

```diff
@@ -82,6 +82,98 @@ a failing test which you then can make pass with a patch to the OSSEC rules:
 .. _syscheck: https://ossec-docs.readthedocs.io/en/latest/docs/manual/syscheck/index.html
 .. _2134: https://github.com/freedomofpress/securedrop/issues/2134
 
+
+How to add a new OSSEC rule?
+=============================
+
+There are two main files involved in this.
```

Comment:
It would be helpful here to describe the 4 different OSSEC packages maintained here, what they contain, and what they are used for.

Reply:
I think it would be useful to add more supporting OSSEC overview docs for developers, like what 4 packages are involved, but not sure if this should fall under the "How to add a new OSSEC rule" section. @kushaldas if you have time, perhaps you can find a better place to include this suggestion as part of this PR, but I think it would also make sense to create a new issue to do a full review of this section of our docs (https://github.com/freedomofpress/securedrop-docs/blob/HEAD/docs/development/updating_ossec.rst) to fix anything that is unclear, out-dated, or missing.